package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.75.1",
+  "version": "2.75.1-fix-for-dl-regressions",
```

Comment:
Please do not overwrite version with version

Reply:
Resolved.